grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcService.java

```diff
@@ -264,7 +264,11 @@ public void messageRead(ByteBufOrStream message) {
                     }
 
                     @Override
-                    public void endOfStream() {}
+                    public void endOfStream() {
+                        if (!res.isDone()) {
```

Comment:
Can you update the comment above or remove it? It was wrong since the reality was 0-1 message, not exactly 1.

Reply:
Thanks! Updated.